tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -557,6 +559,54 @@ def test_update_slurm_reconfigure_race_condition(
         )
 
 
+@pytest.mark.usefixtures("region", "os", "instance", "scheduler")
+def test_slurm_overrides(
+    scheduler,
+    region,
+    pcluster_config_reader,
+    s3_bucket_factory,
+    clusters_factory,
+    test_datadir,
+    scheduler_commands_factory,
+):
+    """Test that run-instances and create-fleet overrides is behaving as expected."""
+
+    bucket_name = s3_bucket_factory()
+    bucket = boto3.resource("s3", region_name=region).Bucket(bucket_name)
+    bucket.upload_file(str(test_datadir / "launch_override.sh"), "launch_override.sh")
+
+    cluster_config = pcluster_config_reader(bucket_name=bucket_name)
+    cluster = clusters_factory(cluster_config)
+    remote_command_executor = RemoteCommandExecutor(cluster)
+    scheduler_commands = scheduler_commands_factory(remote_command_executor)
+
+    # submit two jobs in the two partitions using run-instances and create-fleet overrides.
+    scheduler_commands.submit_command("sleep 1", partition="fleet")
+    scheduler_commands.submit_command("sleep 1", partition="single")
+
+    # Wait slurm_resume and instances to start
+    time.sleep(60)
```

Comment:
Is it possible to use some of the utility functions such as [wait_for_num_instances_in_cluster](https://github.com/aws/aws-parallelcluster/blob/6d80856440a4efe7c15065dbd4a7d9487a4966bd/tests/integration-tests/tests/common/assertions.py#L126-L126) or [wait_for_compute_nodes_states](https://github.com/aws/aws-parallelcluster/blob/6d80856440a4efe7c15065dbd4a7d9487a4966bd/tests/integration-tests/tests/common/hit_common.py#L71-L71) to save on waiting time?

Reply:
done

Reply:
done, thanks!